src/gluonts/model/deepvar_hierarchical/_estimator.py

```diff
@@ -232,6 +230,15 @@ def __init__(
         use_marginal_transformation = False
         conditioning_length = 0
 
+        # This estimator doesn't work in symbolic mode.
+        if trainer.hybridize:
+            logger = logging.getLogger("gluonts").getChild("model")
```

Comment:
Elsewhere we follow the convention of creating a logger at the module level, named after the module, see eg https://github.com/awslabs/gluon-ts/blob/e7e06dd97029efe5f0368a39604e2572f98008a2/src/gluonts/shell/train.py#L32